cogboard-app/src/main/kotlin/com/cognifide/cogboard/config/model/Board.kt

```diff
@@ -1,11 +1,26 @@
 package com.cognifide.cogboard.config.model
 
+import com.fasterxml.jackson.annotation.JsonAnyGetter
+import com.fasterxml.jackson.annotation.JsonAnySetter
+import com.fasterxml.jackson.annotation.JsonIgnore
+
 data class Board(
     val id: String,
     val title: String,
-    val columns: Int,
+    val columns: Int = 8,
     val autoSwitch: Boolean = false,
     val switchInterval: Int,
     val widgets: List<String>,
-    val theme: String
-)
+    val theme: String,
+    val type: String = "WidgetBoard",
+    @JsonIgnore
+    val dynamicFields: MutableMap<String, Any> = mutableMapOf()
+) {
+    @JsonAnySetter
+    fun setFields(name: String, value: Any) {
+        dynamicFields[name] = value
+    }
+
+    @JsonAnyGetter
+    fun getFields() = dynamicFields
+}
```

Comment:
I don't quite understand what is going on here could you please explain?

Reply:
We have discussed this with Szymon and agrred it can stay like this for now (as same solution is used within Widget.kt class), however proper investigation regarding simplifying/refactoring this has to be performed within separate task: https://jira.cognifide.com/jira/browse/CLAN6-266

Reply:
Thanks Michał